macros/src/attr/variant.rs

```diff
@@ -13,6 +13,8 @@ pub struct VariantAttr {
     pub inline: bool,
     pub skip: bool,
     pub untagged: bool,
+    // `docs` is parsed, but JSDocs don't work on unions, therefore it's value is not used
+    pub docs: Vec<String>,
```

Comment:
We should probably just remove this field. If ever support is added, we can come back to it